Utilities/build-script-helper.py

```diff
@@ -107,9 +107,9 @@ def get_swiftpm_options(swift_exec: str, args: argparse.Namespace) -> List[str]:
 
     if platform.system() == 'Darwin':
         swiftpm_args += [
-            # Relative library rpath for swift; will only be used when /usr/lib/swift
-            # is not available.
+            '-Xlinker', '-rpath', '-Xlinker', '/usr/lib/swift',
```

Comment:
Why do we need this one? It's already there by default when I build.

Reply:
For reasons I don’t fully understand it’s not present if we build in CI. I also see the rpath when I build locally. I suspect that it’s because CI is still running macOS 11.5.2, but I haven’t verified that.